src/js/components/Calendar/Calendar.js

```diff
@@ -40,7 +40,12 @@ const headingPadMap = {
 const buildStartEnd = (reference, firstDayOfWeek) => {
   let start = new Date(reference);
   start.setDate(1); // first of month
-  start = subtractDays(start, start.getDay() - firstDayOfWeek); // beginning of week
+
+  start =
+    start.getDay() === 0 && firstDayOfWeek === 1
+      ? (start = subtractDays(start, 6)) // in case Sunday is the first day of the month, and the user asked for Monday to be the first day of the week, we need to include Sunday and six days prior.
```

Comment:
I'm wondering why `prettier` didn't wrap this comment line. We try to keep everything within 80 characters wide, to keep it easy to read in a wide variety of contexts.

Reply:
Anything I can do on my end to enforce it?

Reply:
Reading about this, I've found out that `prettier` does not wrap comments only code.
A workaround I can offer you is to add a rule to .eslintrc to enforce max line length to 80 and you can mark it as error `"max-len": [2, 80, 4]` or warning `"max-len": [1, 80, 4]` depending on your architecture decision.

Reply:
True. I've tested it locally and it seems that only lines of code are being wrapped above 80 characters. and not comments. 
I fixed to comment positioning of this PR and it is good to go. 
@shabtaisharon feel free to file a PR with your suggested fix, adding this rule might be a good addition to grommet, thanks for your inputs!